src/tokens/functional/color/light/app-checks-light.json5

```diff
@@ -0,0 +1,367 @@
+{
+  checks: {
+    bgColor: {
+      $value: '{base.color.gray.9}',
+      $type: 'color',
+    },
+    fgColor: {
+      primary: {
+        $value: '{base.color.gray.0}',
+        $type: 'color',
+      },
+      secondary: {
+        $value: '{base.color.gray.4}',
+        $type: 'color',
+      },
+      link: {
+        $value: '{base.color.blue.3}',
+        $type: 'color',
+      },
+    },
+    button: {
+      iconColor: {
+        rest: {
+          $value: '{base.color.gray.1}',
+          $type: 'color',
+        },
+        hover: {
+          $value: '{fgColor.default}',
+          $type: 'color',
+        },
+      },
+      bgColor: {
+        hover: {
+          $value: '{base.color.gray.2}',
+          $type: 'color',
+          alpha: 0.12,
+        },
+      },
+    },
+    input: {
+      fgColor: {
+        rest: {
+          $value: '{base.color.gray.1}',
+          $type: 'color',
+        },
+        placeholder: {
+          $value: '{base.color.gray.4}',
+          $type: 'color',
+        },
+      },
+      bgColor: {
+        $value: '{base.color.gray.8}',
+        $type: 'color',
+      },
+    },
+    donut: {
+      fgColor: {
+        error: {
+          $value: '{base.color.red.4}',
+          $type: 'color',
+        },
+        pending: {
+          $value: '{base.color.yellow.4}',
+          $type: 'color',
+        },
+        success: {
+          $value: '{base.color.green.4}',
```

Comment:
@langermank I am just working on a PR to update the green and came across all those uses of green.4, etc.

I think in cases like here, where it makes sense, we should use `'{color.success.emphasis}'` and similar instead of referencing a base token.